torch/csrc/utils/python_arg_parser.h

```diff
@@ -287,6 +292,18 @@ inline at::Scalar PythonArgs::scalar(int i) {
   return scalar_slow(i);
 }
 
+inline std::vector<at::Scalar> PythonArgs::scalarlist(int i) {
```

Comment:
The tensor version of this tests if `args[i]` is null, and if it is returns an empty vector. Should you do that here too?

Reply:
Done